documentation/src/main/markdown/releasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Release notes
 
+## Version 8.8.0
+
+### New features
+
+* New Binary Stateless and Container Stateless Scans have been added to [solution_name]. These scans require the new detect.scaaas.scan.path property to be set to either a binary file or a compressed Docker image. These scan types also require a Black Duck Binary Analysis (BDBA) license. See the [Stateless Scans page](downloadingandrunning/statelessscan.md) for further details.
```

Comment:
Perhaps reverse the last two lines and make use of the DITA "Note" object.
* New Binary Stateless and Container Stateless Scans have been added to [solution_name]. These scans require the new detect.scaaas.scan.path property to be set to either a binary file or a compressed Docker image. See the [Stateless Scans page](downloadingandrunning/statelessscan.md) for further details.
<note type="attention">A Black Duck Binary Analysis (BDBA) license is required to execute these scan types.</note> 

The note will look like:
![note](https://user-images.githubusercontent.com/36172712/229613745-d9c7201c-a99c-4243-81b9-0d897b9c06eb.PNG)